src/services/api/__test__/DatagouvfrAPI.test.js

```diff
@@ -0,0 +1,34 @@
+import { http, HttpResponse } from 'msw'
+import { setupServer } from 'msw/node'
+import { afterEach, afterAll, beforeAll, test, expect, vi } from 'vitest'
+
+import { createTestingPinia } from '@pinia/testing'
+import config from '@/config'
+
+import DatagouvfrAPI from '@/services/api/DatagouvfrAPI'
+
+// FIXME: remove once API dependencies to stores are cleaned up
+// create a fake pinia because current API code refers to stores
+createTestingPinia({ createSpy: vi.fn })
```

Comment:
As we don't actually care what goes on within the store, couldn't we just create the testing store without the spy?

Reply:
Probablement, c'est un copy-paste trouvé sur le net, je n'ai pas cherché au delà